framework/src/testing/app_env.ts

```diff
@@ -104,11 +103,7 @@ export class ApplicationEnv {
 		// As we can call this function with different configuration
 		// so we need to make sure existing schemas are already clear
 		codec.clearCache();
-		// TODO: Remove this dependency in future
-		if (!appConfig.modules.includes(DPoSModule)) {
-			appConfig.modules.push(DPoSModule);
-		}
```

Comment:
So now with this change, if someone initializes AppEnv with empty modules `[]` as input and then start application with `AppEnv.startApplication()` how will the application start (and/or forge blocks) without `DPos` module ?